packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3220,6 +3220,7 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
     }
     final _SemanticsFragment fragment = _getSemanticsForParent(
       mergeIntoParent: _semantics?.parent?.isPartOfNodeMerging ?? false,
+      blockUserInteractions: _semantics?.isUserActionsBlocked ?? false,
```

Comment:
UserActions are plural, so it would be `areUserActionsBlocked`.

Also, can we standardize on either UserActions or UserInteractions?